src/config/translation.js

```diff
@@ -40,7 +40,8 @@ const tsObj = {
     'Detach SSH Key successfully.': '解除SSH密钥的绑定成功。',
     'Delete SSH Key successfully.': '删除SSH密钥成功。',
 
-    "Kubernetes can't be selected with others": '不能与其它云服务商一起选择Kubernetes',
```

Comment:
Why create this file,  why not use `translation.json` ?

Reply:
This code is in under ，IDE automatic formatting.